admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -264,6 +270,31 @@ See `Configuring wnd:listen to run as a service
 in the documentation wiki for tips on running the listener as a service via
 cron, and by creating a `systemd`_ startup script.
 
+
+.. _password-options-label:
+
+Password Options
+----------------
+
+There are three ways to supply a password:
+
+#. Interactively in response to a password prompt.
+#. Sent as a parameter to the command, e.g., ``occ wnd:listen host share username password``.
+#. Read from a file, using the ``--password-file`` switch to specify the file to read from. 
+
+.. note::
+   If you use the ``--password-file`` switch, the entire contents of the file will be used for the password, so please be careful with newlines.
+   You can also use ``--password-file=-`` to read the password from the standard input. However, you might need to press ``Ctrl+D`` to close the input and send it to the command.
```

Comment:
I'd skip this case because it's useless. The interesting part is the one about using 3rdparty software to store and fetch the password (which uses that option)